ui/src/dms/MessagesList.tsx

```diff
@@ -66,6 +67,20 @@ export default function MessagesList({
         const group = groups[groupFlag || ''];
         const vessel = group?.fleet[window.our];
         const channel = group?.channels[`chat/${b}`];
+        const club = useChatState.getState().multiDms[b];
+        const dm = useChatState.getState().dms.filter((d) => d === b)[0];
```

Comment:
probably should just use `useMultiDms` and `useDms` outside the loop and reference them here